taprpc/mintrpc/mint.proto

```diff
@@ -172,6 +244,27 @@ message MintingBatch {
     bytes batch_psbt = 7;
 }
 
+message VerboseBatch {
+    /*
+    A public key serialized in compressed format that can be used to uniquely
+    identify a pending minting batch. Responses that share the same key will be
+    batched into the same minting transaction.
+    */
+    bytes batch_key = 1;
+
+    /*
+    The transaction ID of the batch. Only populated if the batch has been
+    committed.
+    */
+    string batch_txid = 2;
+
+    // The state of the batch.
+    BatchState state = 3;
+
+    // The assets that are part of the batch.
+    repeated UnsealedAsset assets = 4;
```

Comment:
Same here re code duplication. Maybe we can also re-use the `MintingBatch` and just have two types of assets (`repeated PendingAsset pending_assets` and `repeated UnsealedAssets unsealed_assets`) of which only ever one is set?